README.md

```diff
@@ -10,6 +10,9 @@ $ VERSION=X.Y.Z make
 
 ## Start local server
```

Comment:
My thought was to keep it separate (and more visible) based on a previous issue requesting an offline version of the docs.
I am still in 2 minds about that reasoning.

Happy to move it under the "Development" section if you would prefer though.